### ISSUE TITLE
Cell Filter?

### ISSUE THREAD

**Issue by hsarji** [holzschu/Carnets#154]
Is there a way to use the Cell Filter nbextension in Carnets? I added tags to some cells. I don't see a way to filter or search for the tags.

**Comment by hsarji** [comment]
I've been looking in Cell Filter. I don't see the search box. I found some info on [Cell filter extension #1240](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/pull/1240#issuecomment-738284471):

> The searchbox should be all to the right in the same toolbar as the Run button and the Code Dropdown. If you are familiar with the developer tools of your browser, you could quickly check if the toolbar still has the same ID.

**Comment by hsarji** [comment]
The Cell Filter nbextension might be broken. *See* [CellFilter exception fails on a jquery statement #154](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/issues/1542).

**Comment by holzschu** [comment]
I've looked at the code. It's pretty old and has not been updated in a while. There's no guarantee it can be fixed.

**Comment by hsarji** [comment]
Ok. I'm new to Jupyter Notebooks. How do extensions make it to the nbextension config menu? It seems that if Cell Filter doesn't work, then it shouldn't be a part of the menu.

**Comment by holzschu** [comment]
I have fixed it. It will be in the next version.
I'm not sure what made it work, but if you want to experiment with your version, you can edit `~/Library/Jupyter/nbextensions/cell_filter/cell_filter.js` and replace the `jqueryui` on line 14 with `jquery`.

(the usual method for editing: copy the file to `~/Documents`, edit outside of Carnets, then copy back into position).

**Comment by hsarji** [comment]
Wow, incredible! Your fix worked for me as well. Cell Filter is very useful. 

Is it possible to add custom-written extensions using the "usual method for editing"?

**Comment by holzschu** [comment]
I think so, yes.

**Comment by hsarji** [comment]
Thank you. I asked about installing custom-written extensions because I wanted to experiment with installing a toolbar button to check the word count. (I mentioned word count in [issue #157](https://github.com/holzschu/Carnets/issues/157). I thought that I just needed to convert [the code](https://github.com/datamics/jupyter-word-count/blob/master/count-jupyter-nb-words.py#L1) from Python to JS. But I noticed that the word count isn't just for one notebook but for the whole directory. So, tweaking it is a bit too challenging for me right now. 

I'm going to close this Cell Filter issue because you fixed it. I'm grateful for your fix, and I'm so happy when using Carnets!
